code/game/machinery/doors/brig_system.dm

```diff
@@ -11,348 +17,333 @@
 
 	var/id = null     		// id of door it controls.
 	var/picture_state		// icon_state of alert picture, if not displaying text/numbers
-	var/list/obj/structure/machinery/targets = list()
+	var/list/obj/structure/machinery/targets = list() // A list of World objects this machine controls.
 	var/list/obj/item/paper/incident/incident_reports = list()
-	var/obj/item/paper/incident/current_report = null
-
-	var/current_menu = "main_menu"
+	var/obj/item/paper/incident/active_report = null // The report currently ticking down the clock.
```

Comment:
i’m aware it’s inconsistent w the rest of the vars here, but if you’re documenting them you might as well dmdoc them so it shows up in vsc hover
```suggestion
/// The report currently ticking down the clock.	var/obj/item/paper/incident/active_report = null```
the same for the vars you’ve touched here